### PR TITLE
Create on on-oci-vm.md

### PR DESCRIPTION
**Pull request by shadabshaukat**
Deployment guide for Airbyte on Oracle Cloud Infrastructure (OCI) VM

## What
OCI (Oracle Cloud Infrastructure) deployment guide for Airbyte

## How
We are created a deployment guide for OCI

.md file created. Please review it

### CODE REVIEW

**Comment by shadabshaukat** [comment]
Hello. Is anyone able to review this documentation?

**Review by marcosmarxm — CHANGES_REQUESTED** [review]
@shadabshaukat Thanks for your contribution and sorry for the delay. LGTM, but could you add the images inside the folder https://github.com/airbytehq/airbyte/tree/master/docs/.gitbook/assets?

**Comment by shadabshaukat** [comment]
@marcosmarxm Thanks Marcos but when i try to upload the image files to "https://github.com/airbytehq/airbyte/tree/master/docs/.gitbook/assets" it tells me "Uploads are disabled, file uploads require push access to this repository"

File: docs/deploying-airbyte/on-oci-vm.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
+Install Airbyte on Oracle Cloud Infrastructure (OCI) VM
+-------------------------------------------------------
```
Comment (avaidyanatha) [inline docs/deploying-airbyte/on-oci-vm.md:2]:
```suggestion
# Install Airbyte on Oracle Cloud Infrastructure (OCI) VM
```

File: docs/deploying-airbyte/on-oci-vm.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+Install Airbyte on Oracle Cloud Infrastructure VM running Oracle Linux 7
+
+Create OCI Instance 
+-------------------
```
Comment (avaidyanatha) [inline docs/deploying-airbyte/on-oci-vm.md:7]:
```suggestion
## Create OCI Instance 
```

File: docs/deploying-airbyte/on-oci-vm.md [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+Go to OCI Console > Compute > Instances > Create Instance
+
+<img width="1665" alt="OCIScreen1" src="https://github.com/airbytehq/airbyte/tree/master/docs/.gitbook/assets/OCIScreen1.png">
+
+<img width="1668" alt="OCIScreen2" src="https://github.com/airbytehq/airbyte/tree/master/docs/.gitbook/assets/OCIScreen2.png">
+
+
+Whitelist Port 8000 for a CIDR range in Security List of OCI VM Subnet
+----------------------------------------------------------------------
```
Comment (avaidyanatha) [inline docs/deploying-airbyte/on-oci-vm.md:16]:
```suggestion
## Whitelist Port 8000 for a CIDR range in Security List of OCI VM Subnet
```

File: docs/deploying-airbyte/on-oci-vm.md [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+Go to OCI Console > Networking > Virtual Cloud Network
+
+Select the Subnet > Security List > Add Ingress Rules
+
+<img width="1423" alt="OCIScreen3" src="https://github.com/airbytehq/airbyte/tree/master/docs/.gitbook/assets/OCIScreen3.png">
+
+
+Login to the Instance/VM with the SSH key and 'opc' user
+--------------------------------------------------------
```
Comment (avaidyanatha) [inline docs/deploying-airbyte/on-oci-vm.md:25]:
```suggestion
## Login to the Instance/VM with the SSH key and 'opc' user
```

File: docs/deploying-airbyte/on-oci-vm.md [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+chmod 600 private-key-file
+
+ssh -i private-key-file opc@oci-private-instance-ip
+
+
+Install docker
+--------------
```
Comment (avaidyanatha) [inline docs/deploying-airbyte/on-oci-vm.md:32]:
```suggestion
### Install Docker
```

File: docs/deploying-airbyte/on-oci-vm.md [hunk 6]
```diff
@@ -0,0 +1,82 @@
...
+
+sudo yum install -y docker
+
+sudo service docker start
+
+sudo usermod -a -G docker $USER
+
+
+Install docker-compose
+----------------------
```
Comment (avaidyanatha) [inline docs/deploying-airbyte/on-oci-vm.md:44]:
```suggestion
### Install Docker Compose
```

File: docs/deploying-airbyte/on-oci-vm.md [hunk 7]
```diff
@@ -0,0 +1,82 @@
...
+
+sudo wget https://github.com/docker/compose/releases/download/1.26.2/docker-compose-$(uname -s)-$(uname -m) -O /usr/local/bin/docker-compose
+
+sudo chmod +x /usr/local/bin/docker-compose
+
+docker-compose --version
+
+
+Install Airbyte
+----------------
```
Comment (avaidyanatha) [inline docs/deploying-airbyte/on-oci-vm.md:54]:
```suggestion
### Install Airbyte
```

File: docs/deploying-airbyte/on-oci-vm.md [hunk 8]
```diff
@@ -0,0 +1,82 @@
...
+<img width="1423" alt="OCIScreen3" src="https://github.com/airbytehq/airbyte/tree/master/docs/.gitbook/assets/OCIScreen3.png">
+
+
+Login to the Instance/VM with the SSH key and 'opc' user
+--------------------------------------------------------
+chmod 600 private-key-file
+
+ssh -i private-key-file opc@oci-private-instance-ip
+
+
```
Comment (avaidyanatha) [inline docs/deploying-airbyte/on-oci-vm.md:30]:
```suggestion
## Install Prerequisites

```

File: docs/deploying-airbyte/on-oci-vm.md [hunk 9]
```diff
@@ -0,0 +1,82 @@
...
+wget https://raw.githubusercontent.com/airbytehq/airbyte/master/{.env,docker-compose.yaml}
+
+which docker-compose 
+
+sudo /usr/local/bin/docker-compose up -d
+
+
+
+Create SSH Tunnel to Login to the Instance
+------------------------------------------
```
Comment (avaidyanatha) [inline docs/deploying-airbyte/on-oci-vm.md:67]:
```suggestion
## Create SSH Tunnel to Login to the Instance

```

File: docs/deploying-airbyte/on-oci-vm.md [hunk 10]
```diff
@@ -0,0 +1,82 @@
...
+
+it is highly recommended to not have a Public IP for the Instance where you are running Airbyte)
+
+From your local workstation
+
+$ ssh -i private-key-file -L 8000:oci-private-instance-ip:8000 opc@bastion-host-public-ip
+
+Access Airbyte
+--------------
```
Comment (avaidyanatha) [inline docs/deploying-airbyte/on-oci-vm.md:76]:
```suggestion
## Access Airbyte
```

**Review by avaidyanatha — COMMENTED** [review]
Just some updates to the headers, but the rest looks great! And yeah, make sure to add those images to the `/assets` folder in the PR.

**Comment by shadabshaukat** [comment]
Thanks for the update on the headers @avaidyanatha  I tried to upload the screen capture images to /assest folder but it gave me message "Uploads are disabled, file uploads require push access to this repository" Could you please help here, thanks

**Comment by marcosmarxm** [comment]
@shadabshaukat you need to add the images in your fork https://github.com/shadabshaukat/airbyte/tree/patch-1/docs/.gitbook/assets probably using the link you should be able to add images, after that you need to change the link in docs

**Comment by marcosmarxm** [comment]
@shadabshaukat one last step is ``

**Comment by shadabshaukat** [comment]
@marcosmarxm Thanks, I have uploaded the images and change the references. Are we good to go ?

**Comment by shadabshaukat** [comment]
@marcosmarxm All done, changed the References to

**Review by marcosmarxm — APPROVED** [review]

**Comment by sherifnada** [comment]
Wow, amazing. Thanks @shadabshaukat !!

**Comment by shadabshaukat** [comment]
Thanks @sherifnada :)  When does the new OCI deployment guide get added to https://docs.airbyte.io/deploying-airbyte

**Comment by sherifnada** [comment]
@shadabshaukat added it here https://github.com/airbytehq/airbyte/pull/4619 -- docs added to the index in `SUMMARY.md` appear on the docs page

**Comment by shadabshaukat** [comment]
Thanks @sherifnada I can see it now in the Airbyte deployment guide. Much appreciated

**Pull request merged**